index.js

```diff
@@ -21,6 +21,7 @@ var errorHandlingStrategies = ['lenient', 'warn', 'throw'];
 module.exports = function (src, options) {
 	options = options || {};
 	options.errorHandlingStrategy = errorHandlingStrategies.indexOf(options.errorHandlingStrategy) !== -1 && options.errorHandlingStrategy || errorHandlingStrategies[0];
+	options.useImageMagick = options.useImageMagick;
```

Comment:
This line was necessary, because the template engine doesn't understand a missing option. According to http://stackoverflow.com/questions/7289916/how-would-you-check-for-undefined-property-in-ejs-for-node-js I can also use `locals.imageMagick` in the `icon.js` file to account for undefined variables.